server/src/main/java/org/opensearch/http/DefaultRestChannel.java

```diff
@@ -69,13 +71,16 @@ public class DefaultRestChannel extends AbstractRestChannel implements RestChann
     static final String CONTENT_TYPE = "content-type";
     static final String CONTENT_LENGTH = "content-length";
     static final String SET_COOKIE = "set-cookie";
+    static final String SERVER_VERSION = "server-version";
+    static final String OPEN_SEARCH_NAME = "OpenSearch";
```

Comment:
This is not need - used only once and in constant expression, could be easily dropped

Reply:
I was told to add these earlier today as static values. See comment above.

Reply:
`SERVER_VERSION` is fine, `OPEN_SEARCH_NAME` is not needed - it is used in exactly 1 constant expression (you could keep it if you want but it has no use)

Reply:
Now its not used at all so it has been removed.